WooCommerce/WooCommerceTests/ViewRelated/Settings/SettingsViewModelTests.swift

```diff
@@ -218,10 +219,28 @@ final class SettingsViewModelTests: XCTestCase {
         XCTAssertFalse(viewModel.sections.contains { $0.rows.contains(SettingsViewController.Row.domain) })
     }
 
-    func test_domain_is_shown_when_domainSettings_feature_is_enabled_and_site_is_wpcom() {
+    func test_domain_is_shown_when_domainSettings_feature_is_disabled_and_site_is_wpcom_for_shop_manager_role() {
```

Comment:
Nit: seems like the test case name is not correct:

```suggestion
    func test_domain_is_not_shown_when_domainSettings_feature_is_enabled_and_site_is_wpcom_for_shop_manager_role() {
```

Reply:
Oops nice catch, fixed in https://github.com/woocommerce/woocommerce-ios/pull/8879/commits/39305c0198c50f917926cb58e72a081665bd1de7